src/amo/containers/App.js

```diff
@@ -57,10 +59,10 @@ export class AppBase extends React.Component {
   }
 
   accountButton() {
-    const { handleLogIn, i18n, isAuthenticated, location } = this.props;
+    const { handleLogIn, handleLogOut, i18n, isAuthenticated, location } = this.props;
     return (
       <button className="button AccountButton"
-              onClick={() => handleLogIn(location)}
+              onClick={() => (isAuthenticated ? handleLogOut() : handleLogIn(location))}
```

Comment:
You encouraged me to make all handler methods `onClick() {}` and then pass off to other methods from there. I'd dig that here, especially so we don't have to use a ternary to set which method to call.